static/copo/js/dtol_functions.js

```diff
@@ -23,7 +23,8 @@ var dt_options = {
         if (
           filter == 'pending' ||
           filter == 'rejected' ||
-          filter == 'bge_pending'
+          filter == 'bge_pending' ||
+          filter == 'associated_project_pending'
```

Comment:
we don't have new tab in the accept_and_reject page, so, no filter called "associated_project_pending"?